lib/operators.js

```diff
@@ -46,46 +46,46 @@
  * @property join
```

Comment:
When replacing this file with a TypeScript file, we don't need this file anymore right?